lnd_test.go

```diff
@@ -6395,9 +6395,20 @@ func testMultiHopHtlcLocalTimeout(net *lntest.NetworkHarness, t *harnessTest) {
 	nodes = []*lntest.HarnessNode{net.Alice}
 	err = lntest.WaitPredicate(func() bool {
 		return assertNumActiveHtlcs(nodes, 0)
-	}, time.Second*15)
+	}, time.Second*8)
 	if err != nil {
-		t.Fatalf("alice's channel still has active htlc's")
+		// It may be the case that due to a race, Bob's sweeping
```

Comment:
I'm not sure if this is the best solution. A call to `waitForTxInMempool` before mining a block should ensure the sweep tx is ready to be mined, then mine the block, then finally `assertTxInBlock` would be a better check.

Reply:
Fixed.